csrc/multidevice/executor.cpp

```diff
@@ -149,7 +149,7 @@ void MultiDeviceExecutor::postKernel(
   }
 }
 
-void MultiDeviceExecutor::postCommunication(SegmentedGroup* group) {
+void MultiDeviceExecutor::postResharding(SegmentedGroup* group) {
```

Comment:
Can you move this to a different PR so merging is easier? I have https://github.com/NVIDIA/Fuser/pull/2172 to merge in conflict with this, so any help to make both PRs incremental is going to be useful!

Reply:
I understand, np. Just be aware that without it there is a collision in the names: `postResharding` as a standalone function and `MultiDeviceExecutor::postResharding`